integration-tests/pom.xml

```diff
@@ -123,6 +123,7 @@
                 <module>class-transformer</module>
                 <module>shared-library</module>
                 <module>hibernate-validator</module>
+                <module>hibernate-validator-resteasy-reactive</module>
```

Comment:
Should this new module be listed in https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json?

Reply:
Maybe let's not add minutes to the native build if we don't think native will introduce issues.

Reply:
That was my though exactly for not adding it